CHANGELOG.md

```diff
@@ -5,6 +5,19 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Added
+
+- Added two new meeting events: `sendingAudioFailed` to indicate a failure in sending audio packets out and `sendingAudioRecovered` to indicate recovery after a `sendingAudioFailed` event
+- Added a new ConnectionHealthPolicy `SendingAudioFailureConnectionHealthPolicy` to evaluate and trigger the above events. Also added `sendingAudioFailureSamplesToConsider` and `sendingAudioFailureInitialWaitTimeMs` to `ConnectionHealthPolicyConfiguration` fields for configuring the new health policy.
+- Added a new field `consecutiveStatsWithNoAudioPacketsSent` in `ConnectionHealthData` for evaluating the `SendingAudioFailureConnectionHealthPolicy`.  
+
+### Changed
+- Refactored `connectionHealthDidChange()` in `MonitorTask` by creating a new `enactHealthPolicy()` method to reduce redundancy in the health policy evaluation.
```

Comment:
nit: `enact` is uncommon vocabulary. `apply`? I guess i see now its a private fn so not a big deal.